README.md

```diff
@@ -48,7 +48,7 @@ or the standalone Mojo SDK:
 - [Get the Mojo SDK](https://docs.modular.com/mojo/manual/get-started/)
 
 Then follow the docs to [write your first Mojo
-program](https://docs.modular.com/mojo/manual/get-started/hello-world).
+program](https://docs.modular.com/mojo/manual/get-started#2-run-code-in-the-repl).
```

Comment:
This is fine, but I'd suggest deleting the above two links and use just one link to the "Get started with Mojo" page, since that already links to the "Install MAX & Mojo" page as the first step.